packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -894,9 +880,11 @@ abstract class DatabaseInstanceSource extends DatabaseInstanceNew implements IDa
  */
 export interface DatabaseInstanceProps extends DatabaseInstanceSourceProps {
   /**
-   * The master user name.
+   * Username and password for the administrative user
+   *
+   * @default - A username of 'admin' and SecretsManager-generated password
    */
-  readonly masterUsername: string;
+  readonly masterUser?: Login;
```

Comment:
Same comment here as in Cluster - how about we come up with a better name for this concept, since we're changing it anyway?